src/site/content/en/blog/idle-detection/index.md

```diff
@@ -83,6 +83,17 @@ if ('IdleDetector' in window) {
 }
 ```
 
+### Polyfilling
+
+Some aspects of the Idle Detection API are polyfillable
+and idle detection libraries like [idle.ts](https://github.com/dropbox/idle.ts) exist,
+but these approaches are constrained to a web app's own content area:
+The library running in the context of the web app
+needs to costly poll for input events or listen to visibility changes.
```

Comment:
```suggestion
needs to expensively poll for input events or listen to visibility changes.
```

Reply:
The adverb form of 'cost' here would be 'costily', but that is not a word in English.

Reply:
I pulled this from the Explainer, but thanks for the note. Expensively works perfectly fine.